react-blogs/Floor-3100/src/components/Card.jsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect } from "react";
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime.js";
+await import('dayjs/locale/es-us.js');
+
+dayjs.extend(relativeTime);
+dayjs.locale('es-us');
+
+// Humanizarfechas
+function humanizarFecha(date) {
+    const wrappedDate = dayjs(date);
+    const daysDiff = Math.abs(wrappedDate.diff(Date.now(), "days"));
+    const currentYear = wrappedDate.year() === dayjs().year();
+    if (!currentYear) {
+        return wrappedDate.format("MMM DD, YYYY")
+    } else if (daysDiff <= 30) {
+        return wrappedDate.fromNow()
+    } else {
+        return wrappedDate.format("MMMM DD")
+    }
+}
+
+const Card = ({ title, desc, img, author, dateTime }) => { 
+
+    useEffect(() => {
+        const timeElement = document.getElementById(dateTime);
+        if (timeElement) {
+            timeElement.innerHTML = "Fecha: " + humanizarFecha(dayjs(timeElement.getAttribute("datetime")));
+        }
+    }, [dateTime]);
+    
+    return (
+                
+        <div className="card">
+            <img src={img} alt={title}/>
+            <hr />
+            <h2><a href="">{title}</a></h2>
+            <p>{desc}</p>
+            <div className="cardBottom">
+                <p className="time"><time id={dateTime} datetime={dateTime}></time></p>
```

Comment:
```suggestion
                <p className="time">{`Fecha: ${humanizarFecha(dayjs(dateTime))}`}</p>
```
Usando las ventajas de react puedes pasar directamente las variables en el render de tu componente, ya no seria necesario usar el useEffect, aunque si quieres dejarlo a modo de practica entiendo